infra/cifuzz/cifuzz.py

```diff
@@ -316,6 +377,7 @@ def check_fuzzer_build(out_dir, sanitizer='address'):
     command += ['-v', '%s:/out' % out_dir]
   command.extend(['-t', 'gcr.io/oss-fuzz-base/base-runner', 'test_all'])
   exit_code = helper.docker_run(command)
+  logging.info('check fuzzer build exit code: %d', exit_code)
```

Comment:
Why log it when no error occurred, maybe only log it in error below.

Reply:
It helps me when i'm debugging because I frequently forget the fact that do_something_fuzzer is explicitly allowlisted by bad_build_check so it doesn't return 0. This is extremely confusing and I've more than once started to try to fix bad build check only to find out it is working as expected. This logging doesn't help a lot but does help a little.